composer.json

```diff
@@ -104,6 +104,9 @@
             "exclude_from_copy": {
                 "packages": [
                     "symfony/deprecation-contracts"
+                ],
+                "file_patterns": [
+                    "/Tests/"
```

Comment:
@jonwaldstein Is this case-sensitive? Do we want to add a `"/tests/"` line as well?

Reply:
Fair enough. I just wasn't sure if adding that would protected further into the future, when new packages are added.

Reply:
Yeah I thought about adding additional checks but it appears this is not common for other packages to include their tests folder, this one just seems to be the outlier - maybe it's not respecting the --no-dev flag or something

Reply:
Ok! Since you looked into it I'm good!